installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations.adoc

```diff
@@ -60,11 +60,24 @@ include::modules/ipi-install-modifying-install-config-for-dual-stack-network.ado
 
 include::modules/configuring-vsphere-regions-zones.adoc[leveloffset=+2]
 
-// begin network customization
+// Network configuration phases
 include::modules/nw-network-config.adoc[leveloffset=+1]
+
+// Specifying advanced network configuration
 include::modules/nw-modifying-operator-install-config.adoc[leveloffset=+1]
+
+// Specifying multiple subnets for your network
+include::modules/nw-operator-vsphere-multiple-subnets.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about the Cluster Network Operator (CNO), see xref:../../../installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations.adoc#nw-operator-cr_installing-vsphere-installer-provisioned-network-customizations[Cluster Network Operator configuration]
+
+* For more information about network configuration parameters for a node, see xref:../../../rest_api/config_apis/infrastructure-config-openshift-io-v1.adoc#spec-platformspec-vsphere-nodenetworking[`.spec.platformSpec.vsphere.nodeNetworking`] in the _API reference_ document.
```

Comment:
Additional resources should only be the link text
```suggestion
* xref:../../../installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations.adoc#nw-operator-cr_installing-vsphere-installer-provisioned-network-customizations[Cluster Network Operator configuration]

* xref:../../../rest_api/config_apis/infrastructure-config-openshift-io-v1.adoc#spec-platformspec-vsphere-nodenetworking[`.spec.platformSpec.vsphere.nodeNetworking`]
```

Reply:
:heavy_check_mark: